modin/pandas/test/test_groupby.py

```diff
@@ -316,6 +316,9 @@ def test_simple_row_groupby(by, as_index, col1_category):
 
     if col1_category:
         pandas_df = pandas_df.astype({"col1": "category"})
+        # As of pandas 1.4.0 operators like min cause TypeErrors to be raised on unordered
```

Comment:
Is this a bug in pandas?

Reply:
I don't think so - according to the pandas docs, trying `min`, `max`, etc. on an unordered categorical column should raise a `TypeError`, and columns are unordered by default, so I think that this is expected behavior?

Reply:
If this is expected behavior, why don't we perform similar thing for Modin?

Reply:
ping

Reply:
Modin behavior is the same, but the test is failing because there was no error before and now the behavior is enforced.

Note that we do `modin_df = from_pandas(pandas_df)` so the dataframes are the same.

Reply:
ok, I see